deploy/runtime/ui/Battery Model.txt

```diff
@@ -503,25 +503,23 @@ function set_batt_defaults_from_data(defs)
 		value('LeadAcid_tn',cell_params.leadacid_tn);
 	}
 	
-	if (cell_params.voltage_choice == 0) {
```

Comment:
This `if` statement needs to be reinstated. The JSON files do not contain both voltage table data and electrochemical data.